pokete_classes/fightmap.py

```diff
@@ -335,15 +325,21 @@ def fight(self, player, enemy, figure, info):
                 obj.attack(attack, enem, self)
             self.show()
             time.sleep(0.5)
+            do_break = False
             if any(i.hp <= 0 for i in players):
                 winner = [i for i in players if i.hp > 0][0]
-                break
+                do_break = True
             elif all(i.ap == 0 for i in obj.attack_obs):
                 winner = [i for i in players if i != obj][0]
                 time.sleep(2)
                 self.outp.outp(f"{obj.ext_name} has used all its' attacks!")
                 time.sleep(3)
-                break
+                do_break = True
+            if do_break:
+                if obj.identifier != "__fallback__" and not obj.player and any(p.hp > 0 for p in figure.pokes[:6]) and ask_bool(self, "Do you want to choose anothoer Pokete?"):
+                    players, player = self.choose_poke(figure, players, player, enemy)
```

Comment:
Ok, this line is WAAY to long!

```suggestion
                if (obj.identifier != "__fallback__"
                    and not obj.player
                    and any(p.hp > 0 for p in figure.pokes[:6])
                    and ask_bool(self,
                                 "Do you want to choose another Pokete?")):
                    players, player = self.choose_poke(figure, players, player,
                                                       enemy)
```